content/academy/apify_platform/getting_started/inputs_outputs.md

```diff
@@ -0,0 +1,88 @@
+
+## [](#accept-input) Accept input into an actor
+
+Let's first create another new actor using the same template as before. Feel free to refer to the [previous lesson]({{@link apify_platform/getting_started/creating_actors.md}}) for a refresher on how to do this.
+
+Replace all of the code in **main.js** with this code snippet:
+
+```JavaScript
+const Apify = require('apify');
+
+Apify.main(async () => {
+    // Grab our numbers which were inputted
+    const { num1, num2 } = await Apify.getInput();
+    
+    // Calculate the solution
+    const solution = num1 + num2;
+
+    // Push the solution to the dataset
+    await Apify.pushData({ solution })
+});
+```
+
+Then, replace everything in **INPUT_SCHEMA.json**  with this:
+
+```JSON
+{
+    "title": "Number adder",
+    "type": "object",
+    "schemaVersion": 1,
+    "properties": {
+        "num1": {
+            "title": "1st Number",
+            "type": "integer",
+            "description": "First number.",
+            "editor": "number"
+        },
+        "num2": {
+            "title": "2nd Number",
+            "type": "integer",
+            "description": "Second number.",
+            "editor": "number"
+        }
+    },
+    "required": ["num1", "num2"]
+}
+```
+
+> If you're interested in learning more about how the code works, and what the **INPUT_SCHEMA.json** means, read about [inputs](https://sdk.apify.com/docs/examples/accept-user-input) and [adding data to a dataset](https://sdk.apify.com/docs/examples/add-data-to-dataset) in the Apify SDK documentation, and refer to the [input schema docs](https://docs.apify.com/actors/development/input-schema#integer).
+
+Finally, **Save** and **Build** the actor just as you did in the previous lesson.
+
+## [](#configuring) Configuring an actor with inputs
+
+If you scroll down a bit, you'll find the **Developer console** located under the multifile editor. By default, after running a build, the **Last build** tab will be selected, where you can see all of the logs related to building the actor. Inputs can be configured within the **Input** tab.
+
+![Configuring inputs]({{@asset apify_platform/getting_started/images/configure-inputs.webp}})
+
+Enter any two numbers you'd like, then press **Start**. The actor's run should complete almost immediately.
+
+## [](#view-results) View actor results
+
+Since we've pushed the result into the default dataset, it, and some info about it can be viewed by clicking this box, which will take you to the results tab:
+
+![Result box]({{@asset apify_platform/getting_started/images/result-box.webp}})
+
+On the results tab, there are a whole lot of options for which format to view/download the data in. Keep the default of **JSON** selected, and click on **Preview**.
+
+![Dataset preview]({{@asset apify_platform/getting_started/images/dataset-preview.webp}})
+
+There's our solution! Did it work for you as well?
```

Comment:
I would mention that it will be stored for the data retention period (link to docs) and that you can retrieve the data via API as well. Actually, API interaction could be one or more lessons of the getting started. So maybe we can just refer to the later lessons here.